backend/blueprint/api/models/user.py

```diff
@@ -6,8 +6,8 @@
 class User(db.Model):
     __tablename__ = 'User'
 
-    id = db.Column(db.String(100), primary_key=True)
-    username = db.Column(db.String(80))
+    id = db.Column(db.String(100))
```

Comment:
why removed `primary_key=True`?

Reply:
user must be recognised by the username and not the id.
Rest is self explanatory.
:smile:

Reply:
@gabru-md Agreed, but why `id` is not an primary key, so every id genrated against `user` must be unique. I guess, I am confused, so can you present some more information supporting the same

Reply:
I guess you are not reading the implementation of the code before creating comments.
I already stated that the answer is self explanatory, but it seems that you cannot understand so I will write here.
The id will be unique no matter what. The id being passed onto the user object is the uid generated by firebase, and which in itself is unique.
I guess you know about removing resundancy, sincr we already have a unique id taken care by firebase, then there is no need to mention it here.

I expect this much explanation is sufficient. :smile: